types/html-to-text/lib/block-text-builder.d.ts

```diff
@@ -25,11 +42,12 @@ export interface BlockTextBuilder {
     /**
      * Add a node inline into the currently built block.
      */
-    addInline(str: string, noWordTransform?: boolean): void;
+    addInline(str: string, options?: AddInlineOptions | boolean): void;
```

Comment:
The boolean here is shortcut for no word transform?

FWIW It could be done like
```ts
addInline(str: string, options?: AddInlineOptions): void;
export type AddInlineOptions = boolean | { noWordTransform?: boolean; };
```

Reply:
I'd like to remind that the top-level boolean is deprecated and I plan to remove it in v9.

Personally I like the current approach that doesn't lift transient things into named types.

Reply:
The boolean is deprecated and will go away at some point.  I had initially coded it as you suggested but changed all of them so there would be fewer changes when the deprecated parameters are removed.

Reply:
Let's do it like this then
```ts
addInline(str: string, options?: AddInlineOptions): void;
/**
 * @deprecated Will be removed in v9. Use options object instead.
 */
addInline(str: string, noWordTransform?: boolean): void;
```

Reply:
I like this suggestion. It will ensure people are more informed and will migrate faster.

Reply:
This gives linting errors for `AddInLine` and `OpenTableCell`:

ERROR: 49:28  unified-signatures  These overloads can be combined into one signature taking `AddInlineOptions | boolean`.

Reply:
I'd just disable that tslint rule. The use case we have is valid. We want one of them annotated as deprecated. tslint isn't taking that into account.